app/columns.tsx

```diff
@@ -6,40 +6,55 @@ import { PreviewLinkButton } from "@/components/ui/previewlinkbutton";
 import { ChevronsUpDown, ChevronDown, ChevronRight } from "lucide-react";
 import { PrettyDateRange } from "./dates";
 import { InfoPopover } from "@/components/ui/infopopover";
-
-function getSurfaceTagColor(template: string): string {
-  const surfaceTagColors: any = {
-    aboutwelcome: "bg-red-400",
-    defaultaboutwelcome: "bg-orange-400",
-    feature_callout: "bg-yellow-300",
-    infobar: "bg-lime-300",
-    milestone_message: "bg-green-400",
-    multi: "bg-teal-300",
-    pb_newtab: "bg-sky-400",
-    protections_panel: "bg-blue-500",
-    toast_notification: "bg-indigo-400",
-    toolbar_badge: "bg-purple-400",
-    spotlight: "bg-pink-400",
-    update_action: "bg-rose-400",
-  };
-
-  if (template in surfaceTagColors) {
-    return surfaceTagColors[template];
-  }
-
-  return "";
-}
+import { getSurfaceDataForTemplate } from "@/lib/messageUtils";
 
 function SurfaceTag(template: string, surface: string) {
+  const tagColors = [
+    "bg-red-400",
+    "bg-orange-400",
+    "bg-yellow-300",
+    "bg-lime-300",
+    "bg-green-400",
+    "bg-teal-300",
+    "bg-sky-400",
+    "bg-blue-500",
+    "bg-indigo-400",
+    "bg-purple-400",
+    "bg-pink-400",
+    "bg-rose-400",
+    "bg-fuchsia-300",
+  ];
+
   let surfaceTagClassName = "";
-  if (template !== "none") {
+  const tagColorIndex = getSurfaceDataForTemplate(template).tagColorIndex;
```

Comment:
So now that we're fetching the `tagColor` from SurfaceData, we won't need to compute the `tagColorIndex` here, we can just directly go

`let surfaceTagClassName = "px-2 py-1 inline rounded-md " + tagColor;`